.gitignore

```diff
@@ -3,6 +3,8 @@ dist
 pdf-work
 kurl.pdf
 
+.idea
```

Comment:
In general we prefer putting things like this in your global gitignore over doing so in-repo